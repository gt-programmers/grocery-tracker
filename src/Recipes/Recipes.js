import React from 'react';
import './Recipes.scss';

const Recipes = () => {
  return (
    <section className="recipes page">
      <p>Recipes</p>
    </section>
  )
}

export default Recipes;