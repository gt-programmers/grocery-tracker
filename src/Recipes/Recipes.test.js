import React from 'react';
import Recipes from './Recipes';
import { shallow } from 'enzyme';

describe("Recipes", () => {
  let wrapper = shallow( <Recipes />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})