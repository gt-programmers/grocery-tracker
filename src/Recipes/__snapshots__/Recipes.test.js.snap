// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipes should match the snapshot 1`] = `
<section
  className="recipes page"
>
  <p>
    Recipes
  </p>
</section>
`;
